guides/v2.3/rest/modules/inventory/manage-low-quantity.md

```diff
@@ -0,0 +1,124 @@
+---
+group: rest-api
+title: Manage low-quantity notifications
+---
+
+The Admin allows the merchant to configure low-quantity notifications from several locations:
+
+* The **Notify for Quantity Below** field (**Stores** > **Configuration** > **Catalog** > **Inventory** > **Product Stock Options**) sets the default value for all products for the entire website/store.
+* The Advanced Inventory **Notify for Quantity Below** field (**Catalog** > **Products** > specific product > **Advanced Inventory**) overrides the value set at the website/store level. The value applies to all of the product's sources. 
+* The **Notify Quantity** fields (**Catalog** > **Products** > specific product > **Assigned Sources** section) override all other settings. The merchant can assign a different threshold for each source for the specific product. Magento deducts these amounts from the available quantity to provide the total salable quantity.
```

Comment:
Maybe move the last sentence up or after the bulleted list to describe the Notify for Quantity Below? Possible revision to consider: "Magento deducts this amount from the total salable quantity for the stock." 
Basically, the amount is deducted from the aggregated amount for all sources assigned to a stock. Side note, unsure you want this here too, if backorders is enabled, this value is not deducted from Salable Quantity.